web/source/dom/domEventHandlers.ts

```diff
@@ -675,9 +675,7 @@ namespace com.keyman.dom {
       // This works OK for iOS, but may need something else for other platforms
       var elem: HTMLElement;
 
-      if(('relatedTarget' in e) && e.relatedTarget) {
-        elem = e.relatedTarget as HTMLElement;
-      }
+      elem = DOMEventHandlers.states._lastActiveElement;
```

Comment:
Is this an unrelated fix? If so, I think it should be in a separate PR for tracking and history.

Reply:
This may have crept in from #7343; I'll double-check.  Work on that one is what led to discovery of this issue, after all.